programmers/naerimchsun.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+    let a = s.length;
+    let b = [];
+    for (let i=0 ; i<a ; i++){
+        b.push(a[i]);
+    }
+    return(b.join("").sort().reverse());
+
+   }

```

Comment:
이 문제는 문자열을 내림차순으로 배치하는 문제입니다.
변수 a에 문자열의 길이를 넣으시고 
배열 b를 선언후에 for문을 이용해서 a[i]로 접근을 하신다면 undefined가 뜹니다. a는 정수형이기 때문입니다.  가희님의 방식은 이렇게 푸실려고 하신것 같습니다.

``` join은 배열을 문자열로 바꿔주는 메소드입니다. sort()는 배열을 정렬하는 메소드입니다. reverse()또한 배열의 원소를 반대로 해주는 메소드입니다. ```
```suggestion
function solution(s) {
    const content = [];
    
    for (let i = 0; i < s.length; i++){
        content.push(s[i]);
    }
    return content.sort().reverse().join("");
}
```

Reply:
배열을 선언할때는 const로 선언해주는 것이 좋습니다. const는 값이 변하지 않지만 속성은 변할 수 있기 때문입니다.

``` return s.split('').sort().reverse().join(''); ``` 
이렇게 한줄로 풀 수도 있습니다. split()은 문자열을 배열로 나눠주는 메소드입니다.